### PR TITLE
Add plugins into sentry metadata

### LINKED ISSUE #1808: Add Plugins into Sentry metadata
__Is your feature request related to a problem? Please describe.__

It'd be nice to see plugins within Sentry errors as we realized all of our existing errors are coming from 3rd party plugins at this point.

__Describe the solution you'd like__

Add Plugins into Sentry metadata.

__Describe alternatives you've considered__

Plugins can be traced by searching for the related EditorID in updates-server logs, however -> Too much work.

### PR DESCRIPTION
**Pull request by barmac**
Closes #1808

### CODE REVIEW

File: app/lib/util/__tests__/error-tracking-spec.js [hunk 1]
```diff
@@ -225,6 +225,54 @@ describe('error-tracking', function() {
...
+
+
+      const calls = rendererSpy.getCalls();
+
+      errorTracking.setTag(Sentry, 'key', 'value');
+
+      calls[0].args[1]();
+
+      // then
+      expect(setTagSpy).to.have.been.calledWith({ key: 'key', value: 'value' });
```
Comment (oguzeroglu) [inline app/lib/util/__tests__/error-tracking-spec.js:273]: Is it maybe better if we go with `calledOnceWith` here? What we really need to see is that setTag method of Sentry is called **only** when `errorTracking.isActive` is true.

Comment (oguzeroglu) [reply app/lib/util/__tests__/error-tracking-spec.js:273]: In other words: We want to make sure `Sentry.configureScope` is not called when `errorTracking.setTag` is called and `errorTracking.isActive` is false.

Comment (barmac) [reply app/lib/util/__tests__/error-tracking-spec.js:276]: I've added a new assertion for that:

https://github.com/camunda/camunda-modeler/pull/1823/commits/4de1e85ee6fc832fcb3d8ffb9a17cd47ff5b9b50#diff-300467e0a9b6480b770adcf53018d2fbR269-R270

Comment (oguzeroglu) [reply app/lib/util/__tests__/error-tracking-spec.js:276]: 👍

Comment (oguzeroglu) [reply app/lib/util/__tests__/error-tracking-spec.js:276]: The tests seem to fail. Could you check?

Comment (barmac) [reply app/lib/util/__tests__/error-tracking-spec.js:276]: yeah 🗡️

Comment (oguzeroglu) [reply app/lib/util/__tests__/error-tracking-spec.js:276]: I think we need to reset errorTracking status within a beforeEach function.

Comment (oguzeroglu) [reply app/lib/util/__tests__/error-tracking-spec.js:276]: By errorTracking status I mean: isActive variable.

Comment (oguzeroglu) [reply app/lib/util/__tests__/error-tracking-spec.js:276]: I suggest either converting errorTracking to a class, or exposing another API (resetStatus) which sets isActive to false and makes additionalTags empty. We'd use this before each test.

Comment (barmac) [reply app/lib/util/__tests__/error-tracking-spec.js:276]: Fixed via https://github.com/camunda/camunda-modeler/pull/1823/commits/29ab6c57827d463d625f99dc9df27ea5009db4f3

**Pull request merged**